### ISSUE TITLE
[a11y] missing label for edit buttons

### ISSUE THREAD

**Issue by susnux** [nextcloud/groupfolders#4467]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

### Steps to reproduce
1. open admin settings
2. see edit buttons
3. inspect accessibility options

### Expected behaviour
Actions have a proper label

### Actual behaviour
No label for those actions

<img width="1751" height="338" alt="Image" src="https://github.com/user-attachments/assets/f1cf2a6b-e282-494e-bcc8-5a0fe8c0d8df" />
